src/angular1-selector.js

```diff
@@ -0,0 +1,137 @@
+/*global window*/
+
+import { Selector, ClientFunction } from 'testcafe';
+
+class Angular1Selector {
+    constructor () {
+        const getComparer            = ClientFunction(exactMatch => {
```

Comment:
You have the wrong auto-formatting because you should add an empty line between multiline definitions
```js
const getComparer = ClientFunction(exactMatch => {
}

const testNodeForAngularAttr = ClientFunction((node, attrSuffix) => {
}

const ensureParent = parentSelector => parentSelector || Selector(() => document.documentElement);
```
instead of 
```js
const getComparer            = ClientFunction(exactMatch => {
}
const testNodeForAngularAttr = ClientFunction((node, attrSuffix) => {
}
const ensureParent           = parentSelector => parentSelector || Selector(() => document.documentElement);
```
Please fix it in whole file.